deps/wazuh_testing/setup.py

```diff
@@ -31,6 +31,7 @@
     'data/qactl_conf_validator_schema.json',
     'data/all_disabled_ossec.conf',
     'data/syscollector_parsed_packages.json',
```

Comment:
Why not remove `syscollector_parsed_packages`? Syscollector packages should generate random packages instead of real ones.